ArchiSteamFarm/Bot.cs

```diff
@@ -3912,6 +3920,154 @@ internal sealed class Bot : IDisposable {
 			return responses.Count > 0 ? string.Join("", responses) : null;
 		}
 
+		private static async Task<string> ResponseTransfer(ulong steamID) {
+			// modifie message / remove?
+			return "To less arguments";
+		}
+
+		private async Task<string> ResponseTransfer(ulong steamID, string mode, string botNameTo) {
+			if ((steamID == 0) || string.IsNullOrEmpty(botNameTo) || string.IsNullOrEmpty(mode)) {
+				ASF.ArchiLogger.LogNullError(nameof(steamID) + " || " + nameof(mode) + " || " + nameof(botNameTo));
+				return null;
+			}
+
+			if (!IsMaster(steamID)) {
+				return null;
+			}
+
+			if (!IsConnectedAndLoggedOn) {
+				return FormatBotResponse(Strings.BotNotConnected);
+			}
+
+			// Not sure if I need to keep it, guess it's better.
+			if (!LootingAllowed) {
+				return FormatBotResponse(Strings.BotLootingTemporarilyDisabled);
+			}
+
+			HashSet<Bot> botsT = GetBots(botNameTo);
```

Comment:
Since you can have only one target, you should not use `GetBots()` here, instead just `Bots.TryGetValue()`.